bip-0340.mediawiki

```diff
@@ -151,25 +151,28 @@ As an alternative to generating keys randomly, it is also possible and safe to r
 Input:
 * The secret key ''sk'': a 32-byte array
 * The message ''m'': a 32-byte array
+* Auxiliary data ''a'': a byte array of length 0 or more
```

Comment:
nit: s/Auxiliary data/Auxiliary rand data?

How about restricting `a`'s length to 32 bytes? Fewer degrees of freedom and easier to provide test vectors.

Reply:
Agree. My thinking was that it could be more than just randomness (e.g. in sign-to-contract additional randomness could go into it as well) - but that's probably something that should be specified separately.